src/shared/dxgi.rs

```diff
@@ -239,6 +239,10 @@ pub const DXGI_PRESENT_USE_DURATION: DWORD = 0x00000100;
 pub const DXGI_PRESENT_ALLOW_TEARING: DWORD = 0x00000200;
 pub const DXGI_ENUM_MODES_INTERLACED: UINT = 1;
 pub const DXGI_ENUM_MODES_SCALING: UINT = 2;
+pub const DXGI_MWA_NO_WINDOW_CHANGES: UINT = 1;
+pub const DXGI_MWA_NO_ALT_ENTER: UINT = 2;
+pub const DXGI_MWA_NO_PRINT_SCREEN: UINT = 4;
```

Comment:
These constants are actually defined as `1 << 0` `1 << 1` and `1 << 2` so can you please preserve that?